core/src/main/java/org/apache/calcite/runtime/CalciteResource.java

```diff
@@ -888,6 +891,15 @@
 
   @BaseMessage("Not a valid input for JSON_STORAGE_SIZE: ''{0}''")
   ExInst<CalciteException> invalidInputForJsonStorageSize(String value);
+
+  @BaseMessage("Not a valid input for JSON_INSERT: jsonDoc: ''{0}'', kvs: ''{1}''")
+  ExInst<CalciteException> invalidInputForJsonInsert(String jsonDoc, String kvs);
```

Comment:
Could you please use "Invalid input" rather than "Not a valid input"? Following are several arguments, not just one.